pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/vedlegg/OpplysningerBruktIBeregningenEndretUttaksgrad.kt

```diff
@@ -0,0 +1,519 @@
+                            }
+                            cell {
+                                includePhrase(KronerText(it))
+                            }
+                        }
+                    }
+
+                    ifNotNull(endretUttaksgradVedVirk.restTilleggspensjon) {
+                        //  vedleggTabellRestTP_001
+                        row {
+                            cell {
+                                text(
+                                    Bokmal to "Tilleggspensjon under opptjening (resttilleggspensjon)",
+                                    Nynorsk to "Tilleggspensjon under opptening (resttilleggspensjon)",
+                                    English to "Remaining supplementary pension"
+                                )
+                            }
+                            cell {
+                                includePhrase(KronerText(it))
+                            }
+                        }
+                    }
+
+                    showIf(beregningKap19VedVirk.forholdstallLevealder.greaterThan(0.0)) {
+                        // vedleggTabellKap19Forholdstall_001
+                        row {
+                            cell {
+                                text(
+                                    Bokmal to "Forholdstall ved uttak",
+                                    Nynorsk to "Forholdstal ved uttak",
+                                    English to "Ratio at withdrawal"
+                                )
+                            }
+                            cell {
+                                eval(beregningKap19VedVirk.forholdstallLevealder.format(scale = 3))
+                            }
+                        }
+                    }
+
+                }
+            }
+        }
+
+        showIf(alderspensjonVedVirk.regelverkType.equalTo(AP2016)) {
+            // vedleggBeregnAP2016Kap20_001
+            paragraph {
+                text(
+                    Bokmal to "For den delen av pensjonen din som er beregnet etter nye regler (kapittel 20) har vi brukt disse opplysningene i beregningen vår:",
+                    Nynorsk to "For den delen av pensjonen din som er berekna etter nye reglar (kapittel 20), har vi brukt desse opplysningane i berekninga vår:",
+                    English to "We have used the following information to calculate the part of your pension that comes under the new provisions (Chapter 20):"
+                )
+            }
+        }
+
+        showIf(alderspensjonVedVirk.regelverkType.isOneOf(AP2016, AP2025)) {
+            paragraph {
+                // vedleggBeregnTabellOverskrift_002
+                table(
+                    header = {
+                        column(columnSpan = 2, alignment = Element.OutlineContent.ParagraphContent.Table.ColumnAlignment.LEFT) {
+                            textExpr(
+                                Bokmal to "Opplysninger brukt i beregningen fra ".expr() + krav.virkDatoFom.format(),
```

Comment:
Mulig dato-formatteringen blir litt annerledes, men det er sikkert fint med full dato i tekst. Kan vurdere å bytte til kort-format for å få plass i kolonnen.

Reply:
Ser ikkje ut som det er nødvendig, og at lengda på denne er godt innanfor.